frontend/assets/l10n/app_en.json

```
@@ -99,7 +99,7 @@
     "codeExpired": "The scanned code has already expired on {{expirationDate}}.",
     "codeInvalid": "The content of the scanned code cannot be understood. It is probably a QR code that was not generated for this app.",
     "codeInvalidMissing": "The content of the scanned code is incomplete. (Error code: {{missing}}Missing)",
-    "codeInvalidType": "The scanned code is not valid for this operation. (Scanned: {{qrCodeType}})",
+    "codeInvalidType": "The scanned code is not valid for activation. (Scanned: {{qrCodeType}}). Please use the QR code shown as activation code.",
```

```suggestion
    "codeInvalidType": "The scanned code is not intended for activation in the app. (Scanned: {{qrCodeType}}). Please use the QR code shown as activation code.",
```
optional suggestion, maybe "not intended" would be more accurate than "not valid"